app/src/main/java/com/sopt/peekabookaos/presentation/recommend/RecommendFragment.kt

```diff
@@ -3,13 +3,81 @@ package com.sopt.peekabookaos.presentation.recommend
 import android.os.Bundle
 import android.view.View
 import com.sopt.peekabookaos.R
+import com.sopt.peekabookaos.data.entity.RecommendData
 import com.sopt.peekabookaos.databinding.FragmentRecommendBinding
 import com.sopt.peekabookaos.util.binding.BindingFragment
-import dagger.hilt.android.AndroidEntryPoint
 
-@AndroidEntryPoint
-class RecommendFragment : BindingFragment<FragmentRecommendBinding>(R.layout.fragment_recommend) {
+class RecommendFragment :
+    BindingFragment<FragmentRecommendBinding>(R.layout.fragment_recommend) {
+    private lateinit var recommendAdapter: BookRecommendAdapter
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        initAdapter()
+    }
+
+    private fun initAdapter() {
+        recommendAdapter = BookRecommendAdapter()
+        binding.rvRecommended.adapter = recommendAdapter
+        recommendAdapter.submitList(mocRecommendationList)
+    }
+
+    companion object {
+        private val mocRecommendationList = listOf(
+            RecommendData(
+                "아무튼, 여름",
+                "김하정",
+                "https://blog.kakaocdn.net/dn/cnCXEQ/btqO7jbSdwy/TDPQ03fpqWkRLr1bhNjQm1/img.jpg",
```

Comment:
```suggestion
                "http://image.yes24.com/goods/90365124/XL"
```
더미데이터 사진 다 이걸로 바꿔주세요.